src/components/Navigation.tsx

```diff
@@ -32,8 +31,9 @@ import ProjectPermissionWarning from "pages/projects/ProjectPermissionWarning";
 import { useSettings } from "context/useSettings";
 import type { LxdProject } from "types/project";
 import { isDarkTheme, loadTheme } from "pages/settings/SettingThemeSwitcher";
+import { useSmallScreen } from "context/useSmallScreen";
 
-const isSmallScreen = () => isWidthBelow(620);
+const isSmallScreen = useSmallScreen();
```

Comment:
We can't invoke the hook here on the top level, it must be in a component. This is probably what is breaking CI.